immutables/_map.c

```diff
@@ -529,10 +529,10 @@ _map_dump_format(_PyUnicodeWriter *writer, const char *format, ...)
     int ret;
 
     va_list vargs;
-#ifdef HAVE_STDARG_PROTOTYPES
-    va_start(vargs, format);
-#else
+#if PY_VERSION_HEX < 0x030C00A1 && !defined(HAVE_STDARG_PROTOTYPES)
```

Comment:
I think we could probably just only have the 2 arg format

Reply:
People seem to be building immutables on odd platforms, so I'm reluctant to drop this.